src/components/lib/ConnectWalletDialog.js

```diff
@@ -22,12 +23,33 @@ const injectedConnector = new InjectedConnector({
 	supportedChainIds: SUPPORTED_CHAIN_IDS
 })
 
+const ledgerConnector = new LedgerConnector({
+	// We use the chainId of mainnet here to workaround an issue with the ledgerjs library.
+	// It currently throws an error for the default chainId of 1377 used by Geth/Ganache.
```

Comment:
Our chainId is 1101 though, no?

Reply:
Yah - we may have changed the default for Geth? In either case, they're both >8 bits.

Reply:
Or geth changed theirs. Can't remember for sure.